sources/packages/backend/apps/queue-consumers/test/helpers/testing-modules/testing-modules-helper.ts

```diff
@@ -11,7 +11,7 @@ import {
   overrideImportsMetadata,
 } from "@sims/test-utils";
 import * as Client from "ssh2-sftp-client";
-import { createMock } from "@golevelup/ts-jest";
+import { DeepMocked, createMock } from "@golevelup/ts-jest";
```

Comment:
Not a part of the PR, I have question on line 45
Will this automatically mock the client.put which uploads the file?
`const sshClientMock = createMock<Client>();`

![image](https://user-images.githubusercontent.com/54600590/234999102-10d55212-9d3b-4f42-a9f1-189ca31a3f9d.png)

Reply:
Yes, `createMock<Client>();` is a helper to do a "deep" mock.